modules/odm_georef/src/Georef.cpp

```diff
@@ -25,7 +29,7 @@ GeorefGCP::~GeorefGCP()
 
 void GeorefGCP::extractGCP(std::istringstream &gcpStream)
 {
-    gcpStream >> x_ >> y_ >> z_ >> pixelX_ >> pixelY_ >> image_;
+    gcpStream >> x_ >> y_ >> z_ >> pixelX_ >> pixelY_ >> image_ >> idgcp_;
```

Comment:
I think this would cause some issues if I use a GCP file which does not include IDs.

Reply:
@pierotofy 
The idgcp_ can help on large datasets or can be used to label GCPs over Ortomosaic.
I upload a QGIS image.
We can comment and remove it to maintain compatibility with old gcp_list.txt layout but i think idgcp_ really can help.

Reply:
Indeed! I'm not saying we should remove it, I'm saying that we should support both formats. Some people might not need to label their GCPs.

Reply:
@pierotofy ,

i think a try:{...}  inside
void GeorefGCP::extractGCP(std::istringstream &gcpStream)
could solve the idgcp_ issue keeping both layouts.
What do you think ?

Reply:
Only one way to find out. Could you compile the changes and test the odm_georef module with both varieties?

Reply:
@pierotofy 

thx for your comments.
Ok, I will compile and test, this weekend.